### PR TITLE
Cors Interceptor by API in Gateway

### PR DESCRIPTION
**Pull request by dijalmasilva**
**Describe Feature**
* Created the Cors Interceptor to API's.

### CODE REVIEW

File: heimdall-core/src/main/java/br/com/conductor/heimdall/core/service/InterceptorService.java [hunk 1]
```diff
@@ -40,6 +40,7 @@
 import br.com.conductor.heimdall.core.util.StringUtils;
 import br.com.conductor.heimdall.core.util.TemplateUtils;
 import br.com.twsoftware.alfred.object.Objeto;
+import ch.qos.logback.core.spi.LifeCycle;
```
Comment (marceloaguiarr) [inline heimdall-core/src/main/java/br/com/conductor/heimdall/core/service/InterceptorService.java]: Is this the correct import?

File: heimdall-core/src/main/java/br/com/conductor/heimdall/core/exception/ExceptionMessage.java [hunk 2]
```diff
@@ -147,7 +147,11 @@
 
     AUTHORIZATION_NOT_FOUND(UNAUTHORIZED.value(), "Authorization not found in header", UnauthorizedException.class),
 
-    RESPONSE_TYPE_NOT_FOUND(BAD_REQUEST.value(), "response_type not found", BadRequestException.class);
+    RESPONSE_TYPE_NOT_FOUND(BAD_REQUEST.value(), "response_type not found", BadRequestException.class),
+
+    CORS_INTERCEPTOR_NOT_API_LIFE_CYCLE(BAD_REQUEST.value(), "The CORS Interceptor must of Lifecycle type API", BadRequestException.class),
```
Comment (marceloaguiarr) [inline heimdall-core/src/main/java/br/com/conductor/heimdall/core/exception/ExceptionMessage.java]: "CORS Interceptor only allowed for API LifeCycle"

File: heimdall-core/src/main/java/br/com/conductor/heimdall/core/service/InterceptorService.java (hunk 1 above)
Comment (dijalmasilva) [reply heimdall-core/src/main/java/br/com/conductor/heimdall/core/service/InterceptorService.java]: Resolved!

File: heimdall-core/src/main/java/br/com/conductor/heimdall/core/exception/ExceptionMessage.java (hunk 2 above)
Comment (dijalmasilva) [reply heimdall-core/src/main/java/br/com/conductor/heimdall/core/exception/ExceptionMessage.java]: Resolved!

**Pull request merged**